lms/djangoapps/courseware/views/views.py

```diff
@@ -984,6 +990,21 @@ def course_about(request, course_id):
             'allow_anonymous': allow_anonymous,
         }
 
+        if not is_testing_environment():
+            total_enrollments = get_course_enrollment_count(course_key)
+            languages = get_all_languages_for_course(overview)
+            instructors = get_all_course_instructors(course_key, request=request)
+
+            additional_context = {
+                'total_enrollments': total_enrollments,
+                'languages': languages,
+                'self_paced': overview.self_paced,
+                'effort': overview.effort,
+                'instructors': instructors
+            }
+
+            context.update(additional_context)
```

Comment:
How about moving all this to a helper in our class and then that helper would simply return all this context?

Reply:
I thought about that, but then decided against it.
This will actually be helpful for anyone working on the course_about page to see the complete context variables. Otherwise, they'd have to keep track of multiple files just to see what is being passed.

Reply:
Thats okay I guess, In that case we will have a method maybe named with extra_context in our adg folder which will return the dict and we will simply update it here. Remember we should avoid core changes.

Reply:
I agree with Zeshan and Asad, it will also reduce the imports in the core, so overall it will reduce the core changes.